internal/profile/resource_profile_test.go

```diff
@@ -233,6 +234,37 @@ func TestAccProfile_project(t *testing.T) {
 	})
 }
 
+func TestAccProfile_defaultProfile(t *testing.T) {
+	projectName := acctest.GenerateName(2, "-")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				// Ensure default profile cannot be managed in default project.
+				Config:      testAccProfile_basic("default"),
+				ExpectError: regexp.MustCompile("Cannot import existing profile"),
+			},
+			{
+				// Ensure default profile cannot be managed in project with "features.profiles".
```

Comment:
Please add a corresponding comment to the "default profile can be managed in non-default projects" case.